gulpfile.js

```diff
@@ -27,7 +27,8 @@ gulp.task('mocha', function () {
   return gulp
     .src(['./test/**/*', '!./test/temp'])
     .pipe(mocha({
-      reporter: 'spec'
+      reporter: 'spec',
+      timeout: Infinity
```

Comment:
Why not have a global timeout of `10000`? This is what we have set on the main project, and it seems to be working out fine https://github.com/meanjs/mean/blob/master/gulpfile.js#L237